### PR TITLE
[OPENJDK-3419] Initial changes for JDK11-ELS images

### PR DESCRIPTION
**Pull request by vieiro**
Update github workflow and README.adoc

Signed-off-by: Antonio Vieiro <avieirov@redhat.com>

### CODE REVIEW

**Comment by jerboaa** [comment]
GHA fails with

```
ERROR Descriptor ('openjdk-11-rhel-9.yaml') could not be found on the path, please check your arguments!
```

The descriptor in this patch is `openjdk-11-rhel9.yaml`

**Comment by vieiro** [comment]
Hi @jmtd @sefroberg,
Ready for review now. I've commented out the failing `behave` tests (we may improve them later on).
Apologies for the force pushes (this is something I'm used to in other projects where squashes are not welcome).

**Review by sefroberg — APPROVED** [review]
Assuming we figure out the behave tests next week, I think this looks correct.

File: .github/workflows/openjdk-11-rhel9.yaml [hunk 1]
```diff
@@ -2,9 +2,9 @@ name: UBI9 OpenJDK 11 S2I Image CI
 on: [push, pull_request]
 env:
   LANG: en_US.UTF-8
-  IMAGE: ubi9-openjdk-11
+  IMAGE: openjdk-11-rhel-9
```
Comment (jmtd) [inline .github/workflows/openjdk-11-rhel9.yaml:5]: It would seem this PR proves that this env var definition here is not in use.

Comment (vieiro) [reply .github/workflows/openjdk-11-rhel9.yaml:5]: Very true!

**Pull request merged**